tests/testthat/test-pivot-wide.R

```diff
@@ -1,3 +1,29 @@
+test_that("error if input is not a data.frame", {
+  spec <- tibble::tibble(
+    .name = c("x", "y", "z"),
+    .value = "val",
+    key = c("x", "y", "z")
+  )
+
+  expect_no_error(
+    tibble(key = c("x", "y", "z"), val = 1:3) %>%
+      tidyr::pivot_wider_spec(spec)
+  )
+  expect_snapshot_error(
+    list(key = c("x", "y", "z"), val = 1:3) %>%
+      tidyr::pivot_wider_spec(spec)
+  )
+  # consider removing if we do not want dbplyr in suggests
```

Comment:
I would remove this because it doesn't really add anything to the test coverage — you've already used a list to check that we're getting an information about what the user actually supplied. In other words, you're now starting to test the behaviour of `obj_type_friendly`, which should be out of scope for this test.